### ISSUE TITLE
Error after going back from Safari to the app

### ISSUE THREAD

**Issue by kevinzzz007** [magus/react-native-facebook-login#30]
Hi so after I click login, I go to the browser, I click confirm, and when I re-launch the app, I get the following error in my console:
```
2:02:33 PM] <START> request:/index.ios.bundle?platform=ios&dev=true
[2:02:33 PM] <END>   request:/index.ios.bundle?platform=ios&dev=true (12ms)
[2:02:33 PM] <START> request:/index.ios.map?platform=ios&dev=true
[2:02:33 PM] <END>   request:/index.ios.map?platform=ios&dev=true (129ms)
::1 - - [12/Oct/2015:18:02:33 +0000] "GET /flow/ HTTP/1.1" 404 18 "-" "TestFBLogin7/1 CFNetwork/758.0.2 Darwin/14.5.0"
[2:02:45 PM] <START> request:/index.ios.bundle
Error: Error validating module options: child "platform" fails because ["platform" is required]
    at /Users/RunzeZhao/Desktop/Weekly projects/TestFBLogin7/node_modules/react-native/packager/react-packager/src/lib/declareOpts.js:60:13
    at /Users/RunzeZhao/Desktop/Weekly projects/TestFBLogin7/node_modules/react-native/packager/react-packager/src/Server/index.js:161:20
    at tryCallOne (/Users/RunzeZhao/Desktop/Weekly projects/TestFBLogin7/node_modules/react-native/node_modules/promise/lib/core.js:37:12)
    at /Users/RunzeZhao/Desktop/Weekly projects/TestFBLogin7/node_modules/react-native/node_modules/promise/lib/core.js:103:15
    at flush (/Users/RunzeZhao/Desktop/Weekly projects/TestFBLogin7/node_modules/react-native/node_modules/promise/node_modules/asap/raw.js:50:29)
    at doNTCallback0 (node.js:407:9)
    at process._tickCallback (node.js:336:13)
```
Tried to look into it, it seems I need to type ```platform=ios```, but as you can see from the error message, I already have that...

**Comment by kevinzzz007** [comment]
Here is my code in index.ios.js
```
/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 */
'use strict';

var React = require('react-native');
var FBLogin = require('react-native-facebook-login');
var FBLoginManager = require('NativeModules').FBLoginManager;
var {
  AppRegistry,
  StyleSheet,
  Text,
  View,
} = React;

var Login = React.createClass({
  render: function() {
    var _this = this;
    return (
      <FBLogin style={{ marginBottom: 10, }}
        permissions={["email","user_friends"]}
        onLogin={function(data){
          console.log("Logged in!");
          console.log(data);
          _this.setState({ user : data.credentials });
        }}
        onLogout={function(){
          console.log("Logged out.");
          _this.setState({ user : null });
        }}
        onLoginFound={function(data){
          console.log("Existing login found.");
          console.log(data);
          _this.setState({ user : data.credentials });
        }}
        onLoginNotFound={function(){
          console.log("No user logged in.");
          _this.setState({ user : null });
        }}
        onError={function(data){
          console.log("ERROR");
          console.log(data);
        }}
        onCancel={function(){
          console.log("User cancelled.");
        }}
        onPermissionsMissing={function(data){
          console.log("Check permissions!");
          console.log(data);
        }}
      />
    );
  }
});

var TestFBLogin7 = React.createClass({
  render: function() {
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>
          Welcome to React Native!
        </Text>
        <Text style={styles.instructions}>
          To get started, edit index.ios.js
        </Text>
        <Text style={styles.instructions}>
          Press Cmd+R to reload,{'\n'}
          Cmd+D or shake for dev menu
        </Text>
        <Login/>
      </View>
    );
  }
});

var styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
  instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
  },
});

AppRegistry.registerComponent('TestFBLogin7', () => TestFBLogin7);
```

**Comment by magus** [comment]
What version of react-native are you using? Try quitting everything and making sure you build fresh...
This seems like an issue that is solved by adding platform like you thought and my gut tells me something is odd with your setup which is causing your query string to not be added/recognized correctly.

https://github.com/facebook/react-native/issues/2645

**Comment by kevinzzz007** [comment]
@magus yeah, I got it fixed!  I just needed to add the name of my app to the Facebook app settings page and then it's all good, the issue was more a Facebook server not accepting my url request, thanks a bunch!

**Comment by kevinzzz007** [comment]
@magus to be more specific so that others can fix it too, I had to add my app's Bundle Id, which is found in Xcode to my facebook app settings page, for the example app, I needed to add ```org.magus.example``` as the Bundle ID to the Facebook app, and everything is working

**Comment by magus** [comment]
Ah so you were attempting to use the example with another Facebook app. Yeah, I believe the setup when creating a Facebook alphas you explicitly identify the bundle ID so the mismatch there seems expected. 

If you have any suggestions for a place I can update the documentation for setup, let me know!

**Comment by kevinzzz007** [comment]
@magus so the issue that I encountered was that when I use the example app and go to Safari, it says `the app is still in development`, which happens if the app is just in the dev mode. So I created an dev app myself and changed the app id and things in the `delegate` file, and then forgot to add the bundle ID.  So if the example app is actually in dev mode, I don't think other users can actually log in with the example app, I would recommend that telling people to use their own app, app id, app name, and add bundle ID to the FB app.  This way, the example app should work.
On another note, have you considered implementing a Parse Facebook login for this repo?  I am currently using the Facebook RN SDK to do it, but I don't really see why we can't do it with this repo...again, I really appreciate you maintaining this repo/answering questions, makes my life so much easier :-)
